src/tool_launcher.cpp

```diff
@@ -1299,7 +1312,26 @@ bool adiscope::ToolLauncher::calibrate()
 	network_btn->setText(status);
 
 	if (calib->isInitialized()) {
-		ok = calib->calibrateAll();
+		if(prefPanel->getAttemptTempLutCalib())
+		{
+			//calib->calibrateFromTemperature();
+			selectedDev->infoPage()->setCalibrationStatusLabel(tr("Calibrated from temperature @ ") + "50G");
```

Comment:
What is 50G?

Reply:
It's a stub for temperature based calibration - 50G is a placeholder for 50 degrees :) 
Temperature based calibration will be implemented in the future.
I will comment it out.